pandas/tests/groupby/test_whitelist.py

```diff
@@ -12,7 +12,7 @@
 
 AGG_FUNCTIONS = ['sum', 'prod', 'min', 'max', 'median', 'mean', 'skew',
                  'mad', 'std', 'var', 'sem']
-AGG_FUNCTIONS_WITH_SKIPNA = ['skew', 'mad']
+AGG_FUNCTIONS_WITH_SKIPNA = ['skew']
```

Comment:
Speaking to why I did this - since `mad` uses the `mean` behind the scenes I figured it made sense to rely on `mean` to do as much of the heavy lifting as possible. Unfortunately `mean` doesn't currently handle the `skipna` parameter but I think it's worth addressing within that function and allowing that to pass through to `mad` rather than implementing specifically within `mad`.

As always glad to open an issue for that if you agree on approach